test/inductor/test_torchinductor_opinfo.py

```diff
@@ -560,6 +568,7 @@ def get_contexts(has_rng_op):
                 # with open("test_output.txt", "a") as f:
                 #     print(f"RUNNING OP {op_name} on {device_type} with {dtype}", flush=True, file=f)
                 #     print(f"RUNNING OP {op_name} on {device_type} with {dtype}", flush=True)
+                rtol, atol = _get_tolerances(dtype)
```

Comment:
Why do you loosen the tolerance for all operators rather than using `inductor_override_kwargs` to just lower the tolerance for `atanh`?